src/services/TechRecordsService.ts

```diff
@@ -105,15 +110,7 @@ class TechRecordsService {
     return this.techRecordsDAO.updateSingle(techRecord)
       .then((data: any) => {
         const response = data.Attributes;
-        const vrms = [{vrm: response.primaryVrm, isPrimary: true}];
-        Object.assign(response, {
-          vrms
-        });
-        // Cleaning up unneeded properties
-        delete response.primaryVrm; // No longer needed
```

Comment:
Should these not still be cleaned up?

Reply:
These lines are moved into formatTechRecordItemForResponse function in order to avoid duplication of code.

Reply:
Ah, that was hidden below the fold. Cheers